docs/versionIndex.md

```diff
@@ -52,6 +52,9 @@ backend services have been tested with each released version of ADF.
 -   [Search Chip Autocomplete Input component](content-services/components/search-chip-autocomplete-input.component.md)
 -   [Search Filter Autocomplete Chips component](content-services/components/search-filter-autocomplete-chips.component.md)
 -   [Is Included pipe](content-services/pipes/is-included.pipe.md)
+-   [Search Date Range Advanced Component](content-services/components/search-date-range-advanced.component.md)
```

Comment:
@swapnil-verma-gl this seems incorrect. We have now 6.2.0 version, please add 6.2.0 version and put it there